backend/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "backend"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tokio = { version = "1.16.1", features = ["full"] }
+serde = { version = "1.0.130", features = ["derive"] }
+serde_json = "1.0.68"
+# warp is optimized for tokio
+warp = "0.3.1"
```

Comment:
why is `warp` being added here?

Reply:
Define routes, http requests, process requests and interact with the db

Reply:
that's what `rocket` is for, `warp` is a competitor to rocket lol